parser/functiondefs.json

```diff
@@ -0,0 +1,16 @@
+{
+    "functions": [
+        {
+            "name": "checkFundsAvailability",
+            "operation": "file://myapis/billingapis.json#checkFunds"
+        },
+        {
+            "name": "sendSuccessEmail",
+            "operation": "file://myapis/emailapis.json#paymentSuccess"
+        },
+        {
+            "name": "sendInsufficientFundsEmail",
+            "operation": "file://myapis/emailapis.json#paymentInsufficientFunds"
+        }
+    ]
+}
```

Comment:
Can't we move these files to `testdata` directory?

Reply:
I think "2" could be a better option then because having these files lying in the package smells. At least in the `testdata` directory, we add purpose.